### PR TITLE
remove stack trace and only keep exception message

### PR DESCRIPTION
**Pull request by liyaqin1**

### CODE REVIEW

**Comment by liyaqin1** [comment]
I deployed the latest commit here: https://deploy.pinadmin.com/env/teletraan/integ-docker/deploy/
tested the latest commit here: https://deploy-integ.pinadmin.com/env/teletraan/prod-docker/new_commits/

File: deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/exception/GenericExceptionMapper.java [hunk 1]
```diff
@@ -72,10 +72,12 @@ public Response toResponse(Throwable t) {
                 return Response.serverError().entity(sb.toString()).build();
             } else {
                 sb.append(errorMessage);
+                /*
```
Comment (ankilosaurus) [inline deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/exception/GenericExceptionMapper.java:75]: Please just remove commented out code

Comment (ankilosaurus) [reply deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/exception/GenericExceptionMapper.java:75]: Also, there is another place at line 49 in the file dumping stack trace.

Comment (liyaqin1) [reply deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/exception/GenericExceptionMapper.java:75]: yea, I will remove the comments.
but are you really sure you want me to remove the first branch around line 49?
That is another branch which I have no idea what it is...

Comment (ankilosaurus) [reply deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/exception/GenericExceptionMapper.java]: That's adjacent area which requires similar improvement but If you are not sure, let's have a followup on that. we can resolve this PR.

**Review by ankilosaurus — APPROVED** [review]

**Pull request merged**